src/base/NekRSProblem.C

```diff
@@ -939,11 +941,13 @@ NekRSProblem::writeVolumeDisplacement(const int elem_id,
                                       const std::vector<double> * add)
 {
   mesh_t * mesh = nekrs::entireMesh();
-  nrs_t * nrs = (nrs_t *)nekrs::nrsPtr();
+  auto nrs = nekrs::nrsPtr();
 
   auto vc = _nek_mesh->volumeCoupling();
   int id = vc.element[elem_id] * mesh->Np;
 
+  auto [x, y, z] = nekrs::host_xyz();
```

Comment:
do we need `host_xyz()`, or could we get away with just the `host_x`, `host_y`, and `host_z` functions?